files/cmi_data_portal_override.php

```diff
@@ -0,0 +1,3 @@
+<?php
+   print file_get_contents( "http://childmindinstitute.org/css/cmi_data_portal_override.css" );
+?>
```

Comment:
Use `readfile()` and drop the final `?>` to avoid accidentally appending more stuff to the response. Helps latency and memory usage by not getting the whole file before starting to commit the response. Caching the file and adding client cache headers wouldn't be a bad idea, but not really important at this point.